cartoframes/datasets.py

```diff
@@ -141,7 +144,7 @@ def _rows(self, df, cols, with_lonlat, geom_col):
                 geom = _decode_geom(the_geom_val)
                 if geom:
                     csv_row += 'SRID=4326;{geom}'.format(geom=geom.wkt)
-            if with_lonlat is not None:
+            if with_lonlat is not None and lng_val is not None and lat_val is not None:
```

Comment:
Where are `lng_val` and `lat_val` set? As far as I can see they are always `None`

Reply:
The part where they are being set is not part of the PR... it's just some test started to fail and realized they were not being initialized 🤷‍♂️ 

https://github.com/CartoDB/cartoframes/blob/master/cartoframes/datasets.py#L130-L134